content/en/dora_metrics/setup/deployments.md

```diff
@@ -1,12 +1,15 @@
 ---
-title: How to Set Up Deployment Data for DORA Metrics
+title: Understanding Deployment Data for DORA Metrics
```

Comment:
I do not understand this change. This page is about the setup, not understanding the data